### ISSUE TITLE
Feature Request: Create binstubs for pact-standalone binaries

### ISSUE THREAD

**Issue by mefellows** [pact-foundation/pact-node#83]
AS a user who wants to integrate to the latest Pact CLI features
IT would be useful to be able to easily reference the Pact binaries (e.g. `pact`, `pact-broker` `pact-message` etc.) 
SO that I can use them without having to write JS wrappers, or wait for them to be included in the JS library

**Example use case**

Prior to deployment, we often run the `can-i-deploy` tool to check if it's safe to release either a consumer or a provider. Bash is an appropriate tool for this. It's hard to find the CLI tools if you don't know what you're doing, but it is possible:

package.json:
```
"can-i-deploy:consumer": "$(find ./node_modules -name pact-broker | grep -e 'bin/pact-broker$' | head -n 1) can-i-deploy --pacticipant SNSPactEventConsumer --latest --broker-base-url https://test.pact.dius.com.au --broker-username xxx --broker-password yyy",
```

Typically, I just install the CLI tools into my Docker container and add them to the path. But it would be nice if I could just go:

package.json:
```
can-i-deploy:consumer": "pact-broker can-i-deploy --pacticipant SNSPactEventConsumer --latest --broker-base-url https://test.pact.dius.com.au --broker-username xxx --broker-password yy
```

I imagine the `./node_modules/.bin` dir might look like:

```
ls ./node_modules/.bin
pact_broker -> ./node_modules/@pact-foundation/pact-node/standalone/darwin-1.33.1/bin/pact-broker
pact -> ./node_modules/@pact-foundation/pact-node/standalone/darwin-1.33.1/bin/pact
...

```

**Comment by mefellows** [comment]
Did you want to have a stab at a PR for it?

**Comment by amr-labib** [comment]
Yes, I could try, it would be my first github PR though. I am reading the contribution guidelines now.

**Comment by mefellows** [comment]
That would be great, thanks!

**Comment by amr-labib** [comment]
I am working on getting this done but I found that we will run into a conflict with one of the binaries. When I look at one of my projects with `pact-node` installed, I find a symlink in `./node_modules/.bin`  defined as:

`pact -> '../@pact-foundation/pact-node/bin/pact-cli.js'`

This will conflict with the pact binary in the standalone package.

What would be recommended here? Should I just ignore that one?

**Comment by mefellows** [comment]
I think we should consider naming the pact node binary `pact-node` to reflect the package name, and the other bin stubs are kept consistent with their originals. We'll want Michel's opinion on this on his return but I can't imagine this blocking you too much.

**Comment by amr-labib** [comment]
@mefellows I kind of have this ready to submit for a pull request but I will be waiting for a response on https://github.com/pact-foundation/pact-ruby-standalone/issues/22
I could submit the PR without it after I had tested on windows but the symlinks won't work out of the box for some users (like me).

**Comment by mefellows** [comment]
Awesome - thanks! Sure, get the PR in and we can pop a label on it to avoid merging. At least the PR process can begin.

**Comment by TimothyJones** [comment]
I think this is now fixed by #161 (thanks @mefellows! Much appreciated!)
